config/routes.rb

```diff
@@ -1,6 +1,22 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  default_url_options({ host: '127.0.0.1', port: 3000 })
```

Comment:
what's this use of this? and why this is here?

Reply:
Re-confirm once